data_model/src/query.rs

```diff
@@ -1073,15 +1073,15 @@ pub mod trigger {
     }
 
     impl Query for FindTriggerById {
-        type Output = Trigger<FilterBox, OptimizedExecutable>;
+        type Output = Trigger<FilterBox, Executable>;
```

Comment:
```suggestion
        type Output = Trigger<FilterBox>;
```

I would prefer this if possible? I'm not sure though on how it's used exactly

Reply:
AFAIU that would require refactoring how we handle optimized executables altogether. I'm not against it, but I feel like it should be a separate issue